test/data/testinit.js

```diff
@@ -35,11 +35,12 @@ this.q = function() {
  * @param {String} a - Assertion name
  * @param {String} b - Sizzle selector
  * @param {String} c - Array of ids to construct what is expected
+ * @param {String} d - Context selector, either `undefined` or `#qunit-fixture`
  * @example t("Check for something", "//[a]", ["foo", "bar"]);
  * @result returns true if "//[a]" return two elements with the IDs 'foo' and 'bar'
  */
-QUnit.assert.t = function( a, b, c ) {
-	var f = jQuery( b ).get(),
+function match( a, b, c, d ) {
```

Comment:
I know this isn't your fault, but I'd rather not exacerbate these meaningless parameter names:
* `a` → `message` (cf. [`deepEqual`](http://api.qunitjs.com/deepEqual/))
* `b` → `selector` (cf. [`jQuery`](http://api.jquery.com/jQuery/))
* `c` → `expectedIds` (cf. [`deepEqual`](http://api.qunitjs.com/deepEqual/))
* `d` → `context` (cf. [`jQuery`](http://api.jquery.com/jQuery/))